test/e2e/elasticsearch_token_propagation_test.go

```diff
@@ -213,9 +218,10 @@ func TestTokenSuite(t *testing.T) {
 }
 
 func (suite *TokenTestSuite) bindOperatorWithAuthDelegator() {
+
 	suite.delegationRoleBinding = &rbac.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "jaeger-operator:system:auth-delegator",
+			Name:      namespace + "jaeger-operator:system:auth-delegator",
```

Comment:
Namespace scoping is not an issue, but just curious why this has been necessary? What was the underlying problems that caused these test failures?

Reply:
Ha, you're right, this is not necessary, I just follow the "convention" I saw in other places:

The underlying issue is this:

In order to avoid parsing the oauth form I'm using a service account token, so I configured the oauth-proxy to use delegate authentication.  

For using delegate authentication I need to bind the system:auth-delegator role to the jaeger-operator service account. (https://github.com/jaegertracing/jaeger-operator/blob/master/pkg/autodetect/main.go#L196) This was working because the oauth-proxy used the same service account used by the operator. 

Is no longer the case, In the actual code, the oauth-proxy uses another service account (I assumed created by the operator itself), so I need to bind it to the `system:auth-delegator` role manually.  Now to make this work I need to bind the` jaeger-operator` account and the oauth-proxy account to the `system:auth-delegator` role.

Question:

- Is this intentional? 
- Should the operator do the binding when it creates the oauth-proxy service account and auth-delegator-available is True?


//cc: @pavolloffay

Reply:
@rubenvp8510 Yeah this was changed a while I ago I think and also possible for end user to specify their own account name.

> Should the operator do the binding when it creates the oauth-proxy service account and auth-delegator-available is True?

Could you create an issue related to this? and may cc Juca in case he has some thoughts.

I'll merge this as don't think there is an issue.